dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckerTest.java

```diff
@@ -294,7 +294,7 @@ public void testFindUnresolvedReferences_unusedLzmaClassByGrpc()
       throws RepositoryException, IOException, ClassNotFoundException {
     String bigTableCoordinate = "com.google.cloud:google-cloud-bigtable:jar:0.66.0-alpha";
     List<Path> paths = StaticLinkageChecker.coordinateToClasspath(bigTableCoordinate);
-    Truth.assertThat(paths).hasSize(53);
+    Truth.assertThat(paths).hasSize(51);
```

Comment:
My MacBook says it's 53.
Circle CI says it's 51, missing servlet-api-2.3.jar and javaee-api-5.0-2.jar.